garage/tf/core/cnn.py

```diff
@@ -0,0 +1,96 @@
+"""CNN with tensorflow as the only dependency."""
+
+import tensorflow as tf
+
+
+def cnn(input_var,
+        output_dim,
+        filter_dims,
+        num_filters,
+        stride,
+        name,
+        padding="SAME",
+        max_pooling=False,
+        pool_shape=(2, 2),
```

Comment:
Is this really a sane default (i.e. that's what people usually use) or should we force the user to specify?

Reply:
In many cases people do not use max pooling layers and in that case, pool shape is not needed. What should be a good way for that?